### ISSUE TITLE
WebP support

### ISSUE THREAD

**Issue by michaelrachlitz** [artpolikarpov/fotorama#499]
Hi,

TL;DR
I ran into an issue where fotorama does not recognize webp as images, thus the slider wont load. Has anyone else experienced this issue and has any ideas on how to solve this?

To elaborate
I am implementing Google's PageSpeed for apache, and it works wonders for our images performance in general, but the WebP compressed images is breaking the Fotorama slider unfortunately.

I dont receive any JS issue, so I don't exactly know where in the files yet this limitation is - about to investigate it. However if anyone else have come across this, or know why a WebP image would not work with the slider, please do share :-)

Happy weekends
